pkg/envoy/lds/inmesh.go

```diff
@@ -383,20 +383,18 @@ func (lb *listenerBuilder) getOutboundTCPFilter(upstream service.MeshService) (*
 func (lb *listenerBuilder) getOutboundFilterChainPerUpstream() []*xds_listener.FilterChain {
 	var filterChains []*xds_listener.FilterChain
 
-	upstreamServices := lb.meshCatalog.ListAllowedOutboundServicesForIdentity(lb.serviceIdentity)
-	if len(upstreamServices) == 0 {
+	dstServices := lb.meshCatalog.ListMeshServicesForIdentity(lb.serviceIdentity)
```

Comment:
Would you mind renaming `dstServices` to `upstreamServices` throughout, just for consistency (which keeps regressing for some reason).

Reply:
done!